File: 3555-final-array-state-after-k-multiplication-operations-i/solution.java
class Solution {
    public int[] getFinalState(int[] nums, int k, int multiplier) {
        int min = Integer.MAX_VALUE, pos = 0;
        int j = 0;
        while(j < k){
            for(int i = 0; i < nums.length; i++){
                if(min>nums[i]) {
                    min = nums[i]; 
                    pos = i;
                }
            }
            nums[pos] = min*multiplier;
            min = Integer.MAX_VALUE;
            j++;
        } 
        return nums;
    }
}

File: 3555-final-array-state-after-k-multiplication-operations-i/solution.cpp
class Solution {
public:
    vector<int> getFinalState(vector<int>& nums, int k, int x) {
        int index;


        for(int i = 0; i < k; i++){
            int min = INT_MAX;
            for(int j = 0; j < nums.size(); j++){
                if(nums[j] < min) {
                    min = nums[j];
                    index = j;
                }
            }
            nums[index] *= x ;
        }
        return nums;
    }
};
